pkg/rpc/request/param.go

```diff
@@ -22,11 +22,9 @@ type (
 	// the server or to send to a server using
 	// the client.
 	Param struct {
```

Comment:
No, we can't, because with `type Param json.RawMessage` JSON unmarshaller expects base64-encoded data as input (like it would be done for `type Param []byte`).